### ISSUE TITLE
FGWP102 ZW5 v3.2 - wrong measurements/reports

### ISSUE THREAD

**Issue by heapxor** [AlCalzone/node-zwave-js#4408]
### Is your problem within Home Assistant (Core or Z-Wave JS Integration)?

NO, my problem is NOT within Home Assistant or the ZWave JS integration

### Is your problem within ZWaveJS2MQTT?

YES, BUT a ZWaveJS2MQTT developer has told me to come here

### Checklist

- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there.

### Describe the bug

What causes the bug?
i dont know

What do you observe?
wrong measurements
![image](https://user-images.githubusercontent.com/64676/160453308-59f64f71-09ae-4cbc-baa4-54d7a657e975.png)

What did you expect to happen?
there is a missing V,
electric consumption is reported in Power which should be in V instead of W

more details here>
https://community.home-assistant.io/t/fgwp102-zw5-v3-2-bug-in-reports/406169

### Device information

Manufacturer: fibaro
Model name:  FGWP102 ZW5 v3.2
Node ID in your network: 7

### How are you using `node-zwave-js`?

- [ ] `zwavejs2mqtt` Docker image (latest)
- [ ] `zwavejs2mqtt` Docker image (dev)
- [ ] `zwavejs2mqtt` Docker manually built _(please specify branches)_
- [ ] `ioBroker.zwave2` adapter _(please specify version)_
- [X] `HomeAssistant zwave_js` integration  _(please specify version)_
- [ ] `pkg`
- [ ] `node-red-contrib-zwave-js` _(please specify version, double click node to find out)_
- [ ] Manually built from GitHub _(please specify branch)_
- [ ] Other _(please describe)_

### Which branches or versions?

version:
`node-zwave-js` branch: 
Driver Version: 8.11.9
Server Version: 1.15.0
`zwavejs2mqtt` branch: Current version: 0.37.0

### Did you change anything?

no

### Did this work before?

Don't know, this is a new device

### Attach Driver Logfile

dont think its required here.

**Comment by AlCalzone** [comment]
This seems to be a duplicate of https://github.com/zwave-js/node-zwave-js/issues/4362 which has additional discussion.
TLDR: The device exposes the power twice, once via the Meter CC (not updated) and once via Multilevel Sensor CC (updated automatically). 

FWIW, Power has the correct unit: https://en.wikipedia.org/wiki/Electric_power
> The [SI](https://en.wikipedia.org/wiki/SI) unit of [power](https://en.wikipedia.org/wiki/Power_(physics)) is the [watt](https://en.wikipedia.org/wiki/Watt), one [joule](https://en.wikipedia.org/wiki/Joule) per [second](https://en.wikipedia.org/wiki/Second).

As for the "missing" measurements mentioned in the forum post - the device does not measure or expose those, see [device manual](https://manuals.fibaro.com/content/manuals/en/FGWPEF-102/FGWPEF-102-EN-A-v2.0.pdf), page 9.

**Comment by heapxor** [comment]
@AlCalzone thanks, well just wondering why V / A are missing as cheap 20euro zwave socket do that ;-(

**Comment by markruys** [comment]
We have a simple work around if you use zwavejs2mqtt: use a Parse Send value: https://github.com/zwave-js/zwavejs2mqtt/issues/2626#issuecomment-1236161413. Works like charm!

Parse Send can be found at Devices values configuration. It creates a function that parses the value sent via MQTT.

**Comment by AlCalzone** [comment]
Requires you to use MQTT though. The recommended connection to HA is the websocket API.
